src/components/LeftSideBar.js

```diff
@@ -0,0 +1,141 @@
+import styled from "styled-components";
+
+const LeftSideBar = (props) => {
+  return (
+    <Container>
+      <Card>
+        <UserInfo>
+          <CardBackground />
+          <a>
+            <Photo />
+            <Link>Welcome, there!</Link>
+          </a>
+          <a>
+            <AddPhotoText>Add a photo</AddPhotoText>
+          </a>
+        </UserInfo>
+        <Widget>
+          <a>
```

Comment:
Avoid using <a> tags
The href attribute would trigger a page refresh which would reset the application states. However the link and navlink of react-router doesn't trigger a page refresh. Since React is used to create single page applications most of the time make sure you choose Link or Navlink when working with routing